src/pt/tsukimangas/src/eu/kanade/tachiyomi/extension/pt/tsukimangas/TsukiMangas.kt

```diff
@@ -38,6 +48,40 @@ class TsukiMangas : HttpSource() {
 
     override val supportsLatest = true
 
+    @delegate:SuppressLint("SetJavaScriptEnabled")
+    private val token: String by lazy {
+        val latch = CountDownLatch(1)
+        var token = ""
+        Handler(Looper.getMainLooper()).post {
+            val webView = WebView(Injekt.get<Application>())
+            with(webView.settings) {
+                javaScriptEnabled = true
+                domStorageEnabled = true
+                databaseEnabled = true
```

Comment:
```suggestion
                databaseEnabled = true
                blockNetworkImage = true
```
if it works with this option that is